### PR TITLE
Update UHMM boundaries + Delete UHPP boundaries

### PR DESCRIPTION
**Pull request by theShpuntik**

### CODE REVIEW

File: Boundaries.geojson [hunk 1]
```diff
@@ -441,8 +441,7 @@
...
-  { "type": "Feature", "properties": { "id": "UHPP", "oceanic": "0", "label_lon": "164.3", "label_lat": "57.0", "region": "EMEA", "division": "VATRUS" }, "geometry": { "type": "MultiPolygon", "coordinates": [ [ [ [ 164.0, 62.07 ], [ 177.0, 61.15 ], [ 180.0, 61.366666 ], [ 180.0, 60.0 ], [ 170.2, 54.816667 ], [ 170.0, 54.678333 ], [ 169.0, 54.0 ], [ 159.0, 50.083333 ], [ 150.0, 45.0 ], [ 150.0, 49.0 ], [ 150.0, 52.166667 ], [ 150.0, 56.0 ], [ 153.983333, 58.016667 ], [ 156.0, 58.916667 ], [ 157.0, 59.333333 ], [ 160.166667, 60.583333 ], [ 162.583333, 61.583333 ], [ 164.0, 62.07 ] ] ], [ [ [ -180.0, 61.366666 ], [ -177.751666, 61.366666 ], [ -180.0, 60.0 ], [ -180.0, 61.366666 ] ] ] ] } },
```
Comment (Adrian2k) [inline Boundaries.geojson:445]: Removoal of a FIR also requires the FIR to be removed in VATSpy.dat.
You also have to assosiate all the airports that has UHPP as a FIR to another FIR.

**Review by Adrian2k — CHANGES_REQUESTED** [review]

File: Boundaries.geojson (hunk 1 above)
Comment (Adrian2k) [reply Boundaries.geojson:445]: Ok, I see the changes is done in #608 
But please next time do not seperate changes into two pull requests.

**Review by Adrian2k — APPROVED** [review]

**Comment by NelisV** [comment]
@theShpuntik Please resolve the merge conflict here.

**Review by NelisV — CHANGES_REQUESTED** [review]
@theShpuntik Please resolve the merge conflict in this pr.

**Review by NelisV — APPROVED** [review]

**Pull request merged**